10-inference.Rmd

```diff
@@ -83,124 +87,120 @@ The population consists of all studio apartment rentals in Vancouver, and the po
-library(gridExtra)
-set.seed(1234)
-virtual_box <- tibble(
-  timbit_id = seq(1, 10000, by = 1),
-  flavour = factor(rbinom(10000, 1, 0.63),
-    labels = c("old fashioned", "chocolate")
-  )
-)
+```{r}
+airbnb
 ```
+Suppose the city of Vancouver wants to know how many Airbnb places are listed as entire homes and apartments (rather than as private or shared rooms). Therefore they may want to estimate the true proportion of all Airbnb listings where the "type of place" is listed as "entire home or apartment." Of course, we usually do not have access to the true population, but here let's imagine (for learning purposes) that our data set represents the population of all Airbnb rental listings in Vancouver, Canada. We can find the proportion of listings where `room_type == "Entire home/apt"`.
 
-```{r 11-example-proportions1}
+```{r 11-example-proportions2, echo = TRUE, message = FALSE, warning = FALSE }
 library(tidyverse)
 library(infer)
-virtual_box
-```
-
-From our simulated box, we can see that the proportion of chocolate Timbits is
-0.63. This value, 0.63, is the *population parameter*. Note that this parameter value is usually unknown in real data analysis problems.
 
-```{r 11-example-proportions2, echo = TRUE, message = FALSE, warning = FALSE}
-virtual_box %>%
-  group_by(flavour) %>%
+airbnb %>%
   summarize(
-    n = n(),
-    proportion = n() / 10000
+    n =  sum(room_type == "Entire home/apt"),
+    proportion = sum(room_type == "Entire home/apt") / nrow(airbnb)
   )
 ```
 
-What would happen if we were to buy a box of 40 randomly-selected Timbits and count the number of chocolate Timbits (*i.e.,* take a random sample of size 40 from our Timbits population)? Let's use R to simulate this using our `virtual_box` population. We can do this using the `rep_sample_n` function from the `infer` package. The arguments
-of `rep_sample_n` are (1) the data frame (or tibble) to sample from, and (2) the size of the sample to take.
+```{r 11-population-parameter, echo = F}
+population_proportion <- airbnb %>%
+  summarize(
+    n =  sum(room_type == "Entire home/apt"),
+    proportion = sum(room_type == "Entire home/apt") / nrow(airbnb)
+  ) %>% 
+  pull()
+```
+
+From our data, we can see that the proportion of `Entire home/apt` listings in the data set is `r round(population_proportion,3)`. This value, `r round(population_proportion,3)`, is the *population parameter*. Remember, this parameter value is usually unknown in real data analysis problems.
+
+What would happen if we were to randomly select 40 listings and count the number of entire home/apartment listings (*i.e.,* take a random sample of size 40 from our population)? Let's use R to simulate this using our `airbnb` population. We can do this using the `rep_sample_n` function from the `infer` package. The arguments of `rep_sample_n` are (1) the data frame (or tibble) to sample from, and (2) the size of the sample to take.
```

Comment:
A lot of this chapter (not just here) is kind of unmotivated like this:

"What would happen if I do X?"

there really isn't a storyline here, just a sequence of what ifs. 

I would suggest a change like the following:

 move the "Remember this param value isn't known in data analysis" from the paragraph before to start this paragraph. like "yeah that's neat but we can't compute it. so instead we are forced to do something else. maybe we can approximate with a small number of data? let's try that."